### DISCUSSION TITLE
[Linear] gradient support...

### DISCUSSION THREAD

**Discussion by memecode** [litehtml/litehtml#297] [Ideas]
[Linear gradient example](https://imgur.com/a/oQcoUlX) in my test browser... see the sitemap down the left hand side. (Yes I know the round edges aren't supported yet).

I've hacked in some support for css gradients. Just linear at this point... the radial ones still do my head in. Much more complicated.
Code is here: https://github.com/litehtml/litehtml/commit/d19ab524517b9e2b1aa789be300fb7aa1ead0c1c

Code may need some cleanup... and I may have missed some things. But the general idea is there.

**Comment by memecode** [comment]
Also not a fan of "\_background_gradient\_" but I didn't see any other way of doing it.. it's not a real CSS property.

**Reply by tordex** [reply]
It is ok. I'm using the same way for some other properties.

**Comment by tordex** [comment]
Excellent work! Thank you! I'll add the angle and the length support and push this into the litehtml.

**Reply by tordex** [reply]
Completely redesigned the background drawing with support of gradients. Please check it: #308
